.changeset/five-geese-crash.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': major
+---
+
+Adjust default [View Transition](https://docs.astro.build/en/guides/view-transitions/) animations.
+
+The `transition:animate` value `morph` has been renamed to `initial`. Elements with a `transition:name` directive but no `transition:animate` directive will now default to `fade`.
```

Comment:
```suggestion
The `transition:animate` value `morph` has been renamed to `initial`. Also, this is no longer the default animation.

 If no `transition:animate` directive is specified, your animations will now default to `fade`.
```
I'm not entirely sure this is correct, but it was a mouthful. Is it the case that `fade` is default instead of `initial` ALL the time? The "with a `transition:name` " threw me off. Is it only in this particular case that there IS a transition name? (not every element will have one, because it's not required to set.) What if there is NO transition name, either? Is that a red herring or is that actually important in deciding which animation is the default?

If the above is correct (`fade` is always the default) then there's no need to mention "with a transition:name". If there's a different default with no transition name, then we also need to specify that, and this becomes a little bit more involved of a definition.  :smile: